solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -771,6 +787,36 @@ contract RandomBeacon is Ownable {
         }
     }
 
+    /// @notice Reports unauthorized groups signing. Must provide a valid signature
+    ///         of the sender's address as a message. Successful signature
+    ///         verification means the private key has been leaked and all group
+    ///         members should be punished by slashing their tokens. Group has
+    ///         to be active or expired. Unauthorized signing cannot be reported
+    ///         for a terminated group. In case of reporting unauthorized
+    ///         signing for a terminated group, or when the signature is invalid,
+    ///         function reverts.
+    /// @param signedMsgSender Signature of the sender's address as a message.
+    function reportUnauthorizedSigning(bytes memory signedMsgSender) external {
+        Groups.Group memory group = groups.getGroup(
+            relay.currentRequest.groupId
+        );
+        relay.validateUnauthorizedSigning(signedMsgSender, group);
+
+        uint64 groupId = relay.currentRequest.groupId;
+
+        groups.terminateGroup(groupId);
+
+        address[] memory groupMembers = sortitionPool.getIDOperators(
+            groups.getGroup(groupId).members
+        );
+        staking.seize(
+            authorization.minimumAuthorization,
+            100,
```

Comment:
Shouldn't we use `unauthorizedSigningNotificationRewardMultiplier` here?

Reply:
Yup, that was a whole point, forgot to update it here. c2d3c65